config/hyde.php

```diff
@@ -350,7 +358,7 @@
         // To get started quickly, you can uncomment the defaults here.
         // See the documentation link above for more information.
         'custom' => [
-            // NavItem::forLink('https://github.com/hydephp/hyde', 'GitHub', 200),
+            // NavigationItem::create('https://github.com/hydephp/hyde', 'GitHub', 200),
```

Comment:
This now causes an exception when reading the configuration as it tries to resolve routes way before this happens. Either we need to revert this deterministic behaviour and go back to deferring evaluation, or we need an array constructor like we just added for Authors. I'm going to experiment with the latter, as it could normalize things with reading these from Yaml https://github.com/hydephp/develop/pull/1632

Reply:
Will be resolved through https://github.com/hydephp/develop/pull/1818